src/components/AnchorForAttachmentsOnly/index.native.tsx

```diff
@@ -1,21 +1,19 @@
 import React from 'react';
 import useThemeStyles from '@hooks/useThemeStyles';
-import * as anchorForAttachmentsOnlyPropTypes from './anchorForAttachmentsOnlyPropTypes';
+import AnchorForAttachmentsOnlyTypes from './AnchorForAttachmentsOnlyTypes';
 import BaseAnchorForAttachmentsOnly from './BaseAnchorForAttachmentsOnly';
 
-function AnchorForAttachmentsOnly(props) {
+function AnchorForAttachmentsOnly({source, displayName}: AnchorForAttachmentsOnlyTypes) {
     const styles = useThemeStyles();
     return (
         <BaseAnchorForAttachmentsOnly
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {...props}
+            source={source}
+            displayName={displayName}
```

Comment:
What is the reason of this change? Is possible to keep old logic (spreading props)?

Reply:
I did this change because I saw the rule of no spreading so decided to follow it.
I'll revert this change 👍